hivemind_redis_database/__init__.py

```
@@ -38,19 +236,116 @@ def add_item(self, client: Client) -> bool:
+            if not client_data:
+                LOG.warning(f"Client '{client_id}' not found for removal")
+                return False
+
+            self.redis.delete(item_key)
+            self.redis.decr(counter_key)
+
+            try:
+                client = cast2client(client_data)
+                self.redis.srem(f"{self.index_prefix}:name:{client.name}", client_id)
+                self.redis.srem(f"{self.index_prefix}:api_key:{client.api_key}", client_id)
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)